### PR TITLE
SB-114-bus-icon

### PR DESCRIPTION
**Pull request by henryluan95**
**What does this code do?**

. Created the bus icon from the project's Figma file.

. Implement the icon into the code base.

Render bus icon

### CODE REVIEW

File: src/pages/RouteDetailsDriver/RouteDetailsDriver.jsx [hunk 1]
```diff
@@ -72,13 +73,21 @@ const RouteDetailsDriver = () => {
...
               lng: driverGeoData.longitude,
             }}
+            icon={{
+              url: busIcon,
+              scaledSize: new window.google.maps.Size(32, 32),
+            }}
           />
         )}
         {data?.data &&
           data?.data.map((rider) => (
```
Comment (StudentCZ) [inline src/pages/RouteDetailsDriver/RouteDetailsDriver.jsx:83]: Hey can you explain what this part means? Thank you

Comment (henryluan95) [reply src/pages/RouteDetailsDriver/RouteDetailsDriver.jsx:83]: I didn't write this but here is how I understand it. In general, when you try to access an object nested inside an object, you will receive a "Cannot read property of undefined ..." even though the value exists. The ? operator ( Optional Channing Operator) will stop the error and allows your code to run normally. 

This part is saying: Check if the data object exists within the data object with the conditional changing operator. If it does, map through the data.data and render all the markers for all the riders.

File: src/pages/RouteDetailsDriver/RouteDetailsDriver.jsx [hunk 2]
```diff
@@ -72,13 +73,21 @@ const RouteDetailsDriver = () => {
               lat: driverGeoData.latitude,
               lng: driverGeoData.longitude,
             }}
+            icon={{
+              url: busIcon,
+              scaledSize: new window.google.maps.Size(32, 32),
```
Comment (yhus) [inline src/pages/RouteDetailsDriver/RouteDetailsDriver.jsx:78]: Is 32px the standard size we'll use for icons? Can we factor that out to some common definition/settings?

**Review by yhus — APPROVED** [review]

File: src/pages/RouteDetailsDriver/RouteDetailsDriver.jsx (hunk 2 above)
Comment (henryluan95) [reply src/pages/RouteDetailsDriver/RouteDetailsDriver.jsx]: Updated.

**Review by FioraDeMarco — APPROVED** [review]

**Review by StudentCZ — APPROVED** [review]
I fixed the merge conflicts and everything looks up to date.

**Pull request merged**